From dfa2705100a893bc017d23406daa9a383fbc3c95 Mon Sep 17 00:00:00 2001
From: Lance Fredrickson <lancethepants@gmail.com>
Date: Wed, 5 Jun 2024 14:31:38 -0600
Subject: [PATCH] com32/lib/syslinux/debug.c: add missing stdio.h include.

Fix building syslinux with GCC 14.x

Add missing stdio.h include. Without it results in the following error under GCC 14.x

../../../com32/lib/syslinux/debug.c: In function ‘syslinux_debug’:
../../../com32/lib/syslinux/debug.c:91:5: error: implicit declaration of function ‘printf’ [-Wimplicit-function-declaration]
   91 |     printf("Dynamic debug unavailable\n");

Upstream: dead (last release in 2014)
Signed-off-by: Lance Fredrickson <lancethepants@gmail.com>
---
 com32/lib/syslinux/debug.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/com32/lib/syslinux/debug.c b/com32/lib/syslinux/debug.c
index d9ab863f..e8f53d57 100644
--- a/com32/lib/syslinux/debug.c
+++ b/com32/lib/syslinux/debug.c
@@ -1,6 +1,7 @@
 #include <linux/list.h>
 #include <string.h>
 #include <stdbool.h>
+#include <stdio.h>
 
 #ifdef DYNAMIC_DEBUG
 
-- 
2.39.2

